modules/creating-serverless-apps-yaml.adoc

```diff
@@ -6,10 +6,14 @@
 [id="creating-serverless-apps-yaml_{context}"]
 = Creating serverless applications using YAML
 
-To create a serverless application by using YAML, you must create a YAML file that defines a `Service` object, then apply it by using `oc apply`.
+Creating Knative resources by using YAML files uses a declarative API, which enables you to describe applications declaratively and in a reproducible manner. To create a serverless application by using YAML, you must create a YAML file that defines a `Service` object, then apply it by using `oc apply`.
```

Comment:
For consistency with context bellow I think it should be a "Knative `Service` object"?

Reply:
It's fine either way, doesn't really need Knative

Reply:
It's fine either way, doesn't really need Knative